packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/Diagram.java

```diff
@@ -50,10 +51,20 @@ public final class Diagram implements IRepresentation {
 
     private DiagramLayoutData layoutData;
 
+    private DiagramStyle style;
+
     private Diagram() {
         // Prevent instantiation
     }
 
+    public static Builder newDiagram(String id) {
+        return new Builder(id);
+    }
+
+    public static Builder newDiagram(Diagram diagram) {
+        return new Builder(diagram);
+    }
+
```

Comment:
We try to let those static method last, I suspect that they have been moved by your formatter?